### ISSUE TITLE
no such package when trying to run `bazel build //buildifier` with a clean repo

### ISSUE THREAD

**Issue by cheister** [bazelbuild/buildtools#79]
I just cloned the repo and ran `bazel build //buildifier` which gives me the following error

```
Extracting Bazel installation...
..........
ERROR: /Users/cheister/Development/buildifier/build/BUILD.bazel:4:1: no such package '@org_golang_x_tools//cmd/goyacc': no such package '@io_bazel_rules_go_repository_tools//': Traceback (most recent call last):
	File "/private/var/tmp/_bazel_cheister/bb74d75b8b198c6a8a6a4a726cae2eab/external/io_bazel_rules_go/go/private/go_repositories.bzl", line 85
		_fetch_repository_tools_deps(ctx, goroot, gopath)
	File "/private/var/tmp/_bazel_cheister/bb74d75b8b198c6a8a6a4a726cae2eab/external/io_bazel_rules_go/go/private/go_repositories.bzl", line 54, in _fetch_repository_tools_deps
		ctx.download_and_extract('%s/archive/%s.zip' % (dep.repo,...), <4 more arguments>)
java.io.IOException: Prefix buildifier-81c36a8418cb803d381335c95f8bb419ad1efa27 was given, but not found in the zip and referenced by '//build:parse.y.go_yacc'.
ERROR: Analysis of target '//buildifier:buildifier' failed; build aborted.
INFO: Elapsed time: 48.146s
```

`go version` is `go version go1.8 darwin/amd64`

**Comment by junghoahnsc** [comment]
I had a same issue now.

**Comment by improbafabian** [comment]
Same issue here since last night.

**Comment by pmbethe09** [comment]
Also: https://github.com/bazelbuild/rules_go/issues/361#issuecomment-292446400

There is a fix if you update to rules_go at HEAD, but we are working on the problem of having broken backwards compatibility.

**Comment by pmbethe09** [comment]
Fixed for rules_go "You can upgrade to 0.3.4, 0.4.2, or master to get your build working again."
